### PR TITLE
Refactor for Contao 4.9

### PR DESCRIPTION
**Pull request by koertho**
This PR refactors the version bundle for contao 4.9.

### CODE REVIEW

File: src/ContaoManager/Plugin.php [hunk 1]
```diff
@@ -14,8 +17,10 @@ class Plugin implements BundlePluginInterface
     public function getBundles(ParserInterface $parser)
     {
         return [
-            BundleConfig::create('HeimrichHannot\VersionsBundle\VersionsBundle')
-                ->setLoadAfter(['Contao\CoreBundle\ContaoCoreBundle'])
+            BundleConfig::create(VersionsBundle::class)
```
Comment (Defcon0) [inline src/ContaoManager/Plugin.php:20]: 2 bundles are created? Correct?

**Review by Defcon0 — APPROVED** [review]
Rest looks good 👍

File: src/ContaoManager/Plugin.php (hunk 1 above)
Comment (koertho) [reply src/ContaoManager/Plugin.php:20]: Yeah, VersionsBundle ist the current (legacy) bundle class, the other one is the new one with correct conventions. I kepts the old class cause of bc. We already have such a setup in utils bundle. The next major release will remove this duplicate.

Comment (Defcon0) [reply src/ContaoManager/Plugin.php:20]: Ah, I suspected that. Thanks for clearification.

**Pull request merged**